src/components/BreakoutRoomsEditor/BreakoutRoomsEditor.vue

```diff
@@ -30,6 +30,9 @@
 			<template v-if="!isEditingParticipants">
 				<div class="breakout-rooms-editor__main">
 					<label class="breakout-rooms-editor__caption" for="room-number">{{ t('spreed', 'Number of breakout rooms') }} </label>
+					<p v-if="isInvalidAmount" class="breakout-rooms-editor__error-hint">
+						{{ t('spreed', 'You can create up to 20 breakout rooms.') }}
+					</p>
 					<NcTextField id="room-number"
```

Comment:
shouldn't the min/max from this component already help with that?

Reply:
With `min`/`max` we can use a native method `input.checkValidity()`, which returns `true`/`false` in case amount is out of range. But that is not implemented upstream in the vue-library, there I believe it belongs

Reply:
yeah that's what I mean. The component should handle this already

Reply:
Yeap, once it is fixed from the upstream we can just replace the boolean `this.isInvalidAmount` with output of the native `checkValidity()` because we still need it for disabling the buttons and maybe keeping the hint 😁.